test/check-terminal

```diff
@@ -26,6 +26,18 @@ class TestTerminal(MachineCase):
         m = self.machine
         b.default_user = "admin"
 
+        # XXX - figure out why locale isn't set up properly by default
```

Comment:
We could set this in our setup scripts. virt-builder allows [setting the timezone](http://libguestfs.org/virt-builder.1.html), but locale is a [different matter](https://rwmj.wordpress.com/2014/01/13/tip-set-the-guest-language-in-virt-builder/) altogether.